modin/pandas/test/dataframe/test_binary.py

```diff
@@ -150,16 +150,13 @@ def test_comparison(data, op, other):
     reason="Test is failing because of mismathing of thrown exceptions. See pandas issue #36377",
 )
 @pytest.mark.parametrize("other", ["a"])
-@pytest.mark.parametrize("op", ["ge", "gt", "le", "lt"])
+@pytest.mark.parametrize("op", ["ge", "gt", "le", "lt", "eq", "ne"])
```

Comment:
This test is still marked as xfail, why do these changes need to be in this PR

Reply:
I had to make changes because this test failed. I don't think that it is xfailed because it was ran by CI.

Reply:
I see, thanks!